ACKategoriesCore/RandomExtensions/Date+Random.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension Date: Randomizable {
+    public static func random() -> Date {
+        Date.random(min: 0)
+    }
+
+    /// - Parameters:
+    ///     - min: Minimum `timeIntervalSince1970`
+    ///     - max: Maxium `timeIntervalSince1970`
+    /// - Returns: Random date
+    public static func random(min: Int = 0, max: Int = Int.max) -> Date {
+        return Date(timeIntervalSince1970: TimeInterval(Int.random(in: min...max)))
+    }
```

Comment:
I'd suggest you go for consistency here and remove the `return` keyword from the second function.

Reply:
why not 🆗